### PR TITLE
Fixed website title

### PR DESCRIPTION
**Pull request by mnkhoi**
### Ticket
[Ticket Link](https://trello.com/c/U7uFosTq/67-change-title-to-something-better-like-mcss-rather-than-the-url)

### What is changed / added
- Fixed website title from the link to "MCSS"

### Screenshots
MCSS Title

### CODE REVIEW

**Review by lx1922 — COMMENTED** [review]
Hey there are some new changes to master configs, can you update your branch to match that... thanks!

**Comment by mnkhoi** [comment]
Done. Can you review it over? Thanks.

**Comment by lx1922** [comment]
I'll take a look...

File: .eslintrc.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ module.exports = {
     "operator-linebreak": 0,
     "arrow-body-style": 0,
     "react/no-unescaped-entities": 0,
```
Comment (lx1922) [inline .eslintrc.js:25]: It's better if we avoid changes to the config files

File: next-env.d.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```
Comment (lx1922) [inline next-env.d.ts:2]: same here...

**Pull request closed without merging**